operator-common/src/main/java/io/strimzi/operator/PlatformFeaturesAvailability.java

```diff
@@ -229,6 +229,15 @@ public boolean hasIngressV1() {
         return this.kubernetesVersion.compareTo(KubernetesVersion.V1_19) >= 0;
     }
 
+    /**
+     * Returns true when the Kubernetes cluster has V1 version of the Ingress resource (Kubernetes 1.16 and newer)
+     *
+     * @return True when PodDisruptionBudget V1 is supported. False otherwise.
+     */
+    public boolean hasPodDisruptionBudgetV1() {
+        return this.kubernetesVersion.compareTo(KubernetesVersion.V1_16) >= 0;
```

Comment:
According to https://kubernetes.io/docs/reference/using-api/deprecation-guide/#poddisruptionbudget-v125, the v1 was added in Kube 1.21 => so this should refer to it.

Reply:
ok, right, i set it to `V1_21`